src/custom-layers/CubemapLayer/CubemapLayer.ts

```diff
@@ -356,7 +358,14 @@ class CubemapLayer implements CustomLayerInterface {
    */
   private animateIn() {
     if (this.imageIsAnimating) {
-      return;
+      return Promise.resolve();
+    }
+
+    if (!this.animationActive) {
+      this.currentFadeOpacity = 1.0;
+      this.imageFadeInDelta = 1;
+      this.map.triggerRepaint();
+      return Promise.resolve();
```

Comment:
Is there a reason for returning `Promise.resolve()` in just two of three branches of this method instead (or in addition to) marking the method as `async`? I can understand it below in `RadialGradientLayer` class for making the method always taking at least a tick, but that's not the case here.